src/session.rs

```diff
@@ -149,14 +145,59 @@ pub fn start(config: &Config, initial_request: Option<String>) -> i32 {
     0
 }
 
-/// When server is not running it's better to cancel blocking request.
-/// Because server can take a long time to initialize or can fail to start.
-/// We assume that it's less annoying for user to just repeat command later
-/// than to wait, cancel, and repeat.
-fn cancel_blocking_request(fifo: String) {
-    debug!("Blocking request but LSP server is not running");
-    let command = "lsp-show-error 'language server is not running, cancelling blocking request'";
-    std::fs::write(fifo, command).expect("Failed to write command to fifo");
+/// Tries to send an error to the client about a request that failed to parse.
```

Comment:
I would probably instead have stuffed the hole in our TOML escaping.
But sure, why not, looks fine.